ci/eval/default.nix

```diff
@@ -152,8 +160,16 @@ let
         # Record and print stats on free memory and swap in the background
         (
           while true; do
-            availMemory=$(free -b | grep Mem | awk '{print $7}')
-            freeSwap=$(free -b | grep Swap | awk '{print $4}')
+            ${if stdenv.isDarwin then ''
+              freeBlocks=$(vm_stat | grep free | awk '{ print $3 }' | sed 's/\.//')
+              inactiveBlocks=$(vm_stat | grep inactive | awk '{ print $3 }' | sed 's/\.//')
+              speculativeBlocks=$(vm_stat | grep speculative | awk '{ print $3 }' | sed 's/\.//')
+              availMemory=$((($freeBlocks+$inactiveBlocks+$speculativeBlocks)*$(pagesize)))
+              freeSwap=$(($(sysctl vm.swapusage | awk '{ print $10; }' | sed 's/M//' | sed -E 's/\..+//') * 1024 * 1024))
```

Comment:
Minor nit, the seds can be removed by just asking awk to print the value as an integer

```bash
freeSwap=$(( $(sysctl vm.swapusage | awk '{ printf "%d", $10 }') * 1024 * 1024 ))
```